src/util.ts

```diff
@@ -58,3 +59,13 @@ export function toSafeHost(rawUrl: string): string {
   // should already have thrown in that case.
   return url.domainToASCII(u.hostname) || u.hostname
 }
+
+/**
+ * Expand a path with ${userHome} in the input string
+ * @param input string
+ * @returns string
```

Comment:
Very tiny nit/question: are these valuable with TypeScript?  My knee-jerk reaction is that at best it duplicates the info TypeScript gives and at worse it could get out of sync if someone changes the signature but not the comment.

But, maybe there is some IDE tooling that uses these.

Reply:
I don't have strong opinions on this at all, my default was just to document the function however typescript folks usually do. I'll leave it for now so you don't have to re-review haha.

Reply:
Fair fair!  No worries about re-review for small things like this, by the way.  :)